books/free-programming-books-uk.md

```diff
@@ -40,7 +40,7 @@
 ### Language Agnostic
 
 * [Дизайн-патерни - просто, як двері](http://designpatterns.andriybuday.com) - Андрій Будай
-
+* []
```

Comment:
typo?

Reply:
yes, wanted to add another book, but it wasn't available anymore. Removed it.